components/RemoteLogin.coffee

```diff
@@ -108,7 +127,13 @@ exports.getComponent = ->
 
     # With browser we check the URL also initially
     checkToken window.location.href, c.params, (err, token) ->
-      return if err
+      if err
+        # Send error with timeout so other context-affecting ops have time to finish
```

Comment:
@jonnor right after there is a context update from another store that "OKs" the state, causing the error message flash very briefly